lib/base-compiler.ts

```diff
@@ -3084,7 +3084,7 @@ export class BaseCompiler implements ICompiler {
                 [...this.compiler.demanglerArgs, '-n', '-p'],
                 {input: result},
             );
-            if (!demangleResult.truncated) {
+            if (demangleResult.stdout.length !== 0 && !demangleResult.truncated) {
```

Comment:
Oops - that's not related, but fixes a different glitch I came across along the way.